grep-0011-scheduler.md

```diff
@@ -0,0 +1,124 @@
+# GREP 0011 -- Create new scheduler to increase performance 
```

Comment:
I would like to propose the title "Create new scheduler based on the Criticality Function". There might be someone else who did a thesis on schedulers using some other function, and that way we can separate those.